### PR TITLE
PS-1574 update to latest IM

### PR DESCRIPTION
**Pull request by pivnicek**
https://keboola.atlassian.net/browse/PS-1574

### CODE REVIEW

File: src/Writer/FileWriter.php [hunk 1]
```diff
@@ -158,10 +159,10 @@ private function uploadFile($source, array $config = [])
      */
     public function tagFiles(array $configuration)
     {
-        $reader = new Reader($this->clientWrapper, $this->logger, new NullWorkspaceProvider());
+        $filesStrategy = new LocalFilesStrategy($this->clientWrapper, $this->logger, new NullWorkspaceProvider(), null);
```
Comment (odinuv) [inline src/Writer/FileWriter.php:162]: so the getFiles method is at the wrong place, it's got nothing to do with localfiles

Comment (pivnicek) [reply src/Writer/FileWriter.php:162]: yes, I suppose it's best to make a 11.0.1 :-(

Comment (pivnicek) [reply src/Writer/FileWriter.php:162]: it's in the AbstractFilesStrategy

Comment (pivnicek) [reply src/Writer/FileWriter.php:162]: @odinuv I had a thought when moving it to make it static and pass the Client as an arg.  would you be ok with that?  That way I can put it back into the Reader class

Comment (pivnicek) [reply src/Writer/FileWriter.php:162]: If you want to see what that looks like I did it here https://github.com/keboola/input-mapping/pull/70

Comment (odinuv) [reply src/Writer/FileWriter.php:162]: yep, that's very reasonable

Comment (pivnicek) [reply src/Writer/FileWriter.php]: yeah, that's much better this way

File: composer.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
         "ext-json": "*"
     },
     "scripts": {
-        "tests": "phpunit",
+        "tests": "phpunit --filter AbsWriterWorkspaceTest",
```
Comment (odinuv) [inline composer.json:40]: 🌵

Comment (pivnicek) [reply composer.json:40]: mofo

Comment (odinuv) [reply composer.json]: 🌮

File: src/Writer/FileWriter.php [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 use Keboola\InputMapping\Reader\NullWorkspaceProvider;
 use Keboola\InputMapping\Reader\Reader;
+use Keboola\InputMapping\Reader\Strategy\Files\LocalFilesStrategy;
```
Comment (odinuv) [inline src/Writer/FileWriter.php:7]: unused import

File: tests/Writer/SynapseWriterWorkspaceTest.php [hunk 4]
```diff
@@ -5,6 +5,7 @@
 use Keboola\InputMapping\Reader\Options\InputTableOptionsList;
 use Keboola\InputMapping\Reader\Reader;
 use Keboola\InputMapping\Reader\State\InputTableStateList;
+use Keboola\InputMapping\Reader\WorkspaceProviderInterface;
```
Comment (odinuv) [inline tests/Writer/SynapseWriterWorkspaceTest.php:8]: so is this

**Review by odinuv — APPROVED** [review]
provided that it passes

**Pull request merged**